### DISCUSSION TITLE
Snap to center 1080p

### DISCUSSION THREAD

**Discussion by codefriar** [rxhanson/Rectangle#1446] [Ideas]
This is a bit of specialized ask. I'd love the ability to snap the upper left to a particular coordinate, and set the window geometry to a given size. In other words, I'd love to snap a window to the center 1080p of my monitor. (i've a stupid wide monitor).

**Comment by rxhanson** [comment]
Thanks for the feedback. You can use this terminal command:
https://github.com/rxhanson/Rectangle/blob/main/TerminalCommands.md#add-an-extra-centering-command-with-custom-size

You can use the free [key codes app](https://apps.apple.com/us/app/key-codes/id414568915) to help get the key code & modifier flags, and you'll need to restart the app for the terminal command to take effect.

Alternatively, you can create a custom size & position shortcut in the UI in [Rectangle Pro](https://rectangleapp.com/pro) (paid).
